sys-devel/clang/clang-18.ebuild

```
@@ -0,0 +1,14 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="C language family frontend for LLVM (metapackage)"
+HOMEPAGE="https://llvm.org/"
+
+LICENSE="metapackage"
+SLOT="${PV}"
+
+RDEPEND="
+	sys-devel/llvm:${PV}[llvm_projects_clang]
```

If `llvm_projects_clang` isn't a USE flag for LLVM, then it's an unmigrated ebuild, so we don't want it (assume it's off):
```suggestion
	sys-devel/llvm:${PV}[llvm_projects_clang(-)]
```